feature-libs/quote/components/quote-actions-by-role/quote-actions-by-role.component.ts

```diff
@@ -55,7 +53,7 @@ export class QuoteActionsByRoleComponent implements OnInit, OnDestroy {
   ngOnInit(): void {
     //submit button present and threshold not reached: Display message
     this.quoteDetails$.pipe(take(1)).subscribe((quote) => {
-      const mustDisableAction = quote.allowedActions.find((action) =>
+      const mustDisableAction = quote?.allowedActions?.find((action) =>
```

Comment:
why did you add optional chaining operators here? - quote is guaranteed not to be null or undefined here. also allowedActions are non optional in type quote. please revert.

Reply:
> quote is guaranteed not to be null or undefined here
It is true but 
also allowedActions are non optional in type quote

Reply:
Done!